metricflow/sql/sql_table.py

```diff
@@ -1,35 +1,16 @@
 from __future__ import annotations
 
+from dataclasses import dataclass
 from typing import Optional, Tuple, Union
 
-from dbt_semantic_interfaces.implementations.base import (
-    FrozenBaseModel,
-    PydanticCustomInputParser,
-    PydanticParseableValueType,
-)
 
-
-class SqlTable(PydanticCustomInputParser, FrozenBaseModel):
+@dataclass(frozen=True, order=True)
```

Comment:
Oh nice, thanks! There was some reason we couldn't do this originally and I guess we forgot to clean it up later.